.devcontainer/Dockerfile

```diff
@@ -73,20 +73,19 @@ RUN apt-get update -y && apt-get install -y \
     libssl1.0.0 \
     ursa=0.3.2-1 \
     # Indy SDK
-    libindy=1.15.0~1625-bionic \
-    # ToDo:
-    #  - Necessary updates to this package have not been published yet.
-    #  - This deb is a special build using the code here:
-    #  - https://github.com/ioflo/ioflo/commit/45bcddbf680d22af84469406a04286ff1c79043a
-    #  - This line can be removed once the release containing these changes is available on PyPi
-    python3-ioflo \
+    #libindy=1.15.0~1625-bionic \
+
+    # new ioflo verion 2.0.3 published on PyPi
+    # python3-ioflo \
     # Need to move libursa.so to parent dir
     && mv /usr/lib/ursa/* /usr/lib && rm -rf /usr/lib/ursa
 
 RUN pip3 install -U \
+    # Required by newer verion of python
+    ioflo==2.0.3 \
```

Comment:
This should be unnecessary.  `ioflo` should be installed as a dev dependency when plenum is installed in teh dev container by `"postCreateCommand": "pip install .[tests]",` in the devcontainer.json file.